packages/google_maps_flutter/google_maps_flutter_ios/pubspec.yaml

```diff
@@ -19,7 +19,8 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  google_maps_flutter_platform_interface: ^2.2.1
+  google_maps_flutter_platform_interface:
+    path: ../google_maps_flutter_platform_interface
```

Comment:
This needs to be reverted before this merges, right?  Does the interface need to be published first?

> warning - pubspec.yaml:23:5 - Publishable packages can't have 'path' dependencies. Try adding a 'publish_to: none' entry to mark the package as not for publishing or remove the path dependency. - invalid_dependency

cc @stuartmorgan

Reply:
Yes and yes

Reply:
Done